src/Polar/kernels.jl

```diff
@@ -604,8 +603,8 @@ KA.@kernel function adj_reactive_power_edge_kernel!(
         adj_coef_sin = -cos_val  * adj_inj
         adj_cos_val  = -coef_sin * adj_inj
 
-        adj_aij =   cos_val*adj_cos_val
-        adj_aij += -sin_val*adj_sin_val
+        adj_aij =   coef_cos * cos_val * adj_inj
+        adj_aij +=  coef_sin * sin_val * adj_inj
```

Comment:
I think you could change this to
```
        adj_aij =   cos_val*adj_sin_val
        adj_aij += -sin_val*adj_cos_va
```
That would be like an AD machine would write it.